.circleci/config.yml

```diff
@@ -1,16 +1,23 @@
-version: 2.1
+version: 2
 
 defaults: &defaults
     working_directory: ~/js-threads-client
     docker:
-        - image: circleci/node:10-browsers
+        - image: textile/builder:1.13.1
 jobs:
     lint-test:
         <<: *defaults
         steps:
             - checkout
             - restore_cache:
                 key: dependency-cache-{{ checksum "package-lock.json" }}
+            - run:
+                name: install daemon
+                background: true
+                command: |
+                    cd ~ && mkdir gosrc && cd gosrc
+                    git clone https://github.com/textileio/go-textile-threads.git && cd go-textile-threads
+                    go run daemon/main.go
```

Comment:
the above could be wrapped up in a new docker image instead for this purpose

Reply:
Good idea, create a ticket?